### DISCUSSION TITLE
DietPi-Software | Weechat

### DISCUSSION THREAD

**Discussion by mintpilo** [MichaIng/DietPi#6546] [Software Request]
Hello, I've been using DietPi for a while now quite comfortably for a few home services (Pihole, unbound, VPN, etc) but I'm surprised weechat isn't in the optimized software packages, (or any IRC related services... bouncers?) a common way people use weechat is remotely from an ssh session or using an external relay application, usually using some multiplexer (like tmux or screen) to detach it from the session, (So they can remain connected to IRC networks) Having this setup the "DietPi way" could make it very comfortable, with defaults to make it not log much to the disk itself, and work well with tmux/screen. Or should that just be left to the user to manually configure?

**Comment by MichaIng** [comment]
Many thanks for your suggestion. Sounds like a cap in our software catalogue indeed. I won't find time to implement it myself anytime soon, but would support any PR to implement it.

Do I get it right that it is basically a console IRC chat client?

It would run it in a service like fashion, e.g. started at boot as systemd service within a screen session (as I know it better than tmux). Then we could add a nice CLI command to simply attach to this screen session from any console, having also a desktop shortcut to open a new console with it.
